README.md

```diff
@@ -153,35 +157,35 @@ end
 
 #### 5. Run the specs
 
-Running the consumer spec will generate a pact file in the configured pact dir (spec/pacts by default).
+Running the AnimalServiceClient spec will generate a pact file in the configured pact dir (spec/pacts by default).
 Logs will be output to the configured log dir that can be useful when diagnosing problems.
 
-Of course, the above specs will fail because the client method is not implemented, so next, implement your client methods.
+Of course, the above specs will fail because the Animal Service client method is not implemented, so next, implement your consumer client methods.
 
```

Comment:
Can we drop the word "consumer" here, it's not actually a consumer client, it's a provider client.